src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2385,25 +2385,25 @@
     <value>Preview popup</value>
   </data>
   <data name="ToggleCompactOverlay" xml:space="preserve">
-    <value>Toggle compact overlay</value>
+    <value>Compact overlay</value>
   </data>
   <data name="OpenHelpDescription" xml:space="preserve">
     <value>Open the online help page in your browser</value>
   </data>
   <data name="ToggleFullScreenDescription" xml:space="preserve">
-    <value>Toggle full screen mode</value>
+    <value>Toggle between full screen mode</value>
   </data>
   <data name="EnterCompactOverlayDescription" xml:space="preserve">
-    <value>Enter compact overlay mode</value>
+    <value>Switch to compact overlay mode</value>
```

Comment:
This change is good in theory, but it would be best to revert for consistency with 'Exit compact overlay'.